provision/provision-utility.sh

```diff
@@ -5,7 +5,7 @@
 PROVISIONER="/srv/provision/utilities/${1}/${2}/provision.sh"
 
 if [[ -f $PROVISIONER ]]; then
-    ( source "${PROVISIONER}" )
+    ( bash "${PROVISIONER}" >> "${VVV_CURRENT_LOG_FILE}" )
```

Comment:
This just pipes everything to the file, that's what `log_to_file` was meant to do, is the issue not that `log_to_file` is not doing what it's supposed to do? What about things that appear on `stderr`? Those are meant to appear in the terminal but wouldn't here

Reply:
Let me run some tests, but this is precisely (minus the bash, which was implicit) before #2116 . Check it out here. 

### Before

Log to file

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/f63b32159a949053644589545ae0d0ed9d4133f2/provision/provision-utility.sh#L3-L9

Provisioner execution

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/f63b32159a949053644589545ae0d0ed9d4133f2/provision/provision-utility.sh#L18-L23

Because the provisioners are executable, this will spawn a new bash, and avoid stdout of this script, but append it into the $logfile

### Now on this PR

Log to file logic

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/82f384ba59ced9f0b0c0603adf308445167a74b3/provision/provision-helpers.sh#L131-L142

Provisioner execution

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/82f384ba59ced9f0b0c0603adf308445167a74b3/provision/provision-utility.sh#L8-L10

Like possible to be seen here, i'm just reverting functionality as it was, to avoid outputing information of the actual utility provisioner to the bash, same as before, but still outputing useful information as to timing.